### ISSUE TITLE
iPads and speed

### ISSUE THREAD

**Issue by closdesign** [blasten/turn.js#165]
We are testing on our iPad 1 (don't know if it runs better on the newer iPads) but it seems to be kind of clunky, but it works.  Is there a way to smooth out the transitions? Make it a bit more app like, is there a way to leverage the hardware acceleration a bit more?  I have read, for some reason or another,  that the iPads stop Javascript usage when a transition is being used.  Is there a way to get around that.

This thing works wonders on desktops, but for the tablets, it is a bit slower. Not sure if this is a processor issue either.

Any suggestions?  Thanks.

**Comment by blasten** [comment]
Yes, it runs a lot better on iPad 3. The only thing you can try to speed it up is to disable all the gradients & shadows and hardware acceleration must be on. Add the option `gradients: false` and `acceleration: true`.  http://turnjs.com/docs/Turn_Options

**Comment by closdesign** [comment]
Well that takes out all the fun then.  Thanks man. I appreciate the input.

**Comment by blasten** [comment]
I know, it also depends on the size of the book/magazine and the complexity of the pages.  For example the first website of turnjs.com had a sample that rendered all the shadows pretty fast in iPad 1, in terms of what to expect in a safari mobile browser.

**Comment by closdesign** [comment]
Yeh.   We had someone test on iPad 1 and someone on iPad 3. iPad 3 of course is going to be better just mainly because of the hardware and better processor.  We may just skip iPad 1 and say "oh well" like we do with IE6 now.  Or anything below IE7.  I think it is also the images we are using. Lots of text on the images.  Huge files that I had to try to compress and make them not look like crap to load them faster.  I am using your default 1443px height like the samples. Seems to look pretty well.  I hopefully will have an iPad 3 in-house tomorrow to borrow to see if anything is better.  I personally think everything works great. But I showed someone on the iPad 1 and they thought it was jumpy.  I had to explain the issues with mobile Safari and it issue with the slow process on the iPad.  We are just trying to get a solution that will work on most devices and not have to download a native app and can be cross browser. And your solution, and support, by far has been the best.  
Did you see what the Fantasy Interactive guys did with Glimpse  with version 3 (I am assuming you have). Pretty nice stuff.  Shows how extensible this thing can become.

Thanks again.
